spec/services/attribute_asserter_spec.rb

```diff
@@ -437,11 +311,19 @@
             it 'only includes uuid, email, aal, and ial (no verified_at)' do
               expect(user.asserted_attributes.keys).to eq %i[uuid email aal ial]
             end
+
+            it 'does not create a getter function for IAL1 attributes' do
+              expected_email = EmailContext.new(user).last_sign_in_email_address.email
+              expect(get_asserted_attribute(user, :email)).to eq expected_email
+            end
+
+            it 'gets UUID (MBUN) from Service Provider' do
```

Comment:
I see that MBUN is referenced elsewhere, but what does it mean?

Reply:
I looked it up. I think it is Meaningless But Unique Number. I think that it is confusing and should be removed.

Reply:
Interesting, thanks. I think I'd disagree that it's meaningless too.

Reply:
early SAML docs used it more, [we document it in the glossary](https://handbook.login.gov/articles/glossary.html#mbun)

Reply:
removed in https://github.com/18F/identity-idp/pull/11035/commits/29ea702bba588ba00ca09b4e39a7dde0dd8c78f6 !